src/core/collection/CollectionCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Card from "../card/Card";
+import { CollectionItemType } from "./types";
+
+export default function CollectionCard({
+  id,
+  title,
+  link,
+  featuredImage,
+}: CollectionItemType) {
+  if (!title || !link) {
+    return null;
+  }
+  // TODO: Render a variation of a card depending on which kind of item is being handled
+  // E.g. use props `__typeName: type` or fields to descrbe the type of the collection item.
```

Comment:
Since this is a configurable component, the application can decide itself how it resolves the different card types.
https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/11/files#diff-7c14115b1294c5339759d600404d9c5ef041df250cf92bb4c65b0767ffae585aR26
```
  components: {
    A: (props: React.AnchorHTMLAttributes<HTMLAnchorElement>) => JSX.Element;
    Img: (props: React.ImgHTMLAttributes<HTMLImageElement>) => JSX.Element;
    Link?: (
      props: React.AnchorHTMLAttributes<HTMLAnchorElement>
    ) => JSX.Element;
    Head?: (props: { children: React.ReactNode }) => JSX.Element;
    CollectionCard?: (props: CollectionItemType) => JSX.Element;
  };
```

Reply:
`CollectionItemType` should actually have more information so the decision could be made in `CollectionCard`.